sample/app/TODO.md

```diff
@@ -13,10 +13,8 @@ options:
 -> The swc project will probably exports:
```

Comment:
should this file be removed?

Reply:
I'll remove the file once the Webpack config has been added